### ISSUE TITLE
dependsOn = Mandatory

### ISSUE THREAD

**Issue by kgansberger** [madeyourday/contao-rocksolid-custom-elements#135]
an error occurs if dependsOn is true and field has 'mandatory'=>'true' .

**Comment by ausi** [comment]
Can you provide a small example configuration?

**Comment by kgansberger** [comment]
```
...
'insertImage' => array(
	'label'                   => array('de' => array('Bild einfügen','')),
	'default'               => 0,
	'inputType'          => 'checkbox',
	'eval'                    => array('tl_class'=>'w50'),
),
'image' => array(
	'label'                   => array('Bild wählen',''),
	'inputType'               => 'fileTree',
	'default'                 => 0,
	'dependsOn' => array(
		'field' => 'insertImage',  
		'value' => '1',
	),
	'eval'                    => array('mandatory'=>'true', 'fieldType'=>'radio', 'files'=>true, 'filesOnly'=>true, 'tl_class'=>'clr'), 
        'extensions'=>GLOBALS['TL_CONFIG']['validImageTypes']),
),
...
```

**Comment by ausi** [comment]
Could you please verify if the current master branch works correctly?

```json
"madeyourday/contao-rocksolid-custom-elements": "dev-master as 2.3.999",
```

**Comment by jankout** [comment]
I can confirm this behavior too. :-)

**Comment by ausi** [comment]
> I can confirm this behavior too although I copied your changed javascript.

Did you clear your browser cache afterwards or tried the feature in the inkognito mode of your browser?

**Comment by jankout** [comment]
Yes, I tried it in the incognito mode (Chrome) and unfortunately, it did work. As I mentioned I copied only the new part of your javascript.

**Comment by ausi** [comment]
Please try the current master branch if the issue appears there too:

```
"madeyourday/contao-rocksolid-custom-elements": "dev-master as 2.3.999",
```

**Comment by jankout** [comment]
I tried it. But it didn't work as expected. There is the same error message in the console if I don't fill out the second mandatory field which is currently in the hidden part. 

`An invalid form control with name='vimeo' is not focusable.`

**Comment by ausi** [comment]
Could you please try it again with `"dev-master as 2.3.999"`?
I think your case did’t work because you used `standardField`s, this should be fixed now too.

**Comment by jankout** [comment]
Hello, it seems to work. Thank you very much.

**Comment by ausi** [comment]
Released as version 2.3.5
